.gitignore

```diff
@@ -5,3 +5,4 @@ _site/**
 .sass-cache/**
 CNAME
 .travis.yml
+.idea/
```

Comment:
I like to use idea to work on the markdown text (its actually pretty nice). This makes sure no one accidentally commits idea related files into the repo. Fairly harmless addition but i can remove it if you wish.